vllm/attention/backends/torch_sdpa.py

```diff
@@ -124,10 +124,6 @@ def __init__(
             raise ValueError(
                 f"Head size {head_size} is not supported by PagedAttention. "
                 f"Supported head sizes are: {supported_head_sizes}.")
-        if kv_cache_dtype != "auto":
-            raise NotImplementedError(
-                "Torch SDPA backend does not support FP8 KV cache. "
-                "Please use xFormers backend instead.")
```

Comment:
This is only implemented for CPU usage of the Torch SPDA backend, right? Would fp8 kv cache work if we used CUDA with the SPDA backend, for instance?